base/bin/migration_template.mustache

```diff
@@ -0,0 +1,10 @@
+const r = require('root/lib/db');
+
+module.exports = {
+  up: () => {
+    return Promise.resolve();
+  },
+  down: () => {
+    return Promise.resolve();
+  }
```

Comment:
Those are just placeholders for the person generating the migration file to come in and write their actual code. I expect that the majority of actual migrations would be > 1 line.